app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomPresenter.kt

```diff
@@ -513,6 +529,9 @@ class ChatRoomPresenter @Inject constructor(
                                     isRoom = true
                                 ))
                                 messagesRepository.saveAll(messages.result)
+                                //if success - saving last synced time
+                                // TODO sort and peek the latest date
+                                messagesRepository.saveLastSyncDate(messages.result.last().timestamp)
```

Comment:
and we call messagesRepository.saveLastSyncDate() inside loadMissingMessages() because here we again are in the point of chat history synchronization